src/app/components/pool/action/add/vm-size-picker/vm-size-picker.component.spec.ts

```diff
@@ -32,30 +33,27 @@ describe("VmSizePickerComponent", () => {
     let vmSizeServiceSpy;
     let accountServiceSpy;
     let pricingServiceSpy;
-
+    let passVms;
+    let iaasVms;
     beforeEach(() => {
-        vmSizeServiceSpy = {
-            vmSizeCategories: of({
-                standard: ["Standard_A*"],
-                compute: ["Standard_C*"],
-                memory: ["Standard_M*"],
-            }),
-            virtualMachineSizes: of(List([
-                new VmSize({ name: "Standard_A1" } as any),
-                new VmSize({ name: "Standard_A2" } as any),
-                new VmSize({ name: "Standard_A3" } as any),
-                new VmSize({ name: "Standard_C1" } as any),
-                new VmSize({ name: "Standard_C2" } as any),
-                new VmSize({ name: "Standard_O1" } as any),
-            ])),
-            cloudServiceSizes: of(List([
-                new VmSize({ name: "Standard_A1" } as any),
-                new VmSize({ name: "Standard_A2" } as any),
-                new VmSize({ name: "Standard_A3" } as any),
-                new VmSize({ name: "Standard_C1" } as any),
-                new VmSize({ name: "Standard_C2" } as any),
-            ])),
-        };
+        passVms = [];
+        iaasVms = [];
+        const vmSizeStrings = supportedVms.split("\n").filter(value => !!value.trim());
```

Comment:
that seems a bit unecessary to have this. Why not just have mock data as it was before? It makes it harder to see why test are failing if it loads all that big file